docs/installation.md

```diff
@@ -23,15 +21,25 @@ curl -fsSL https://apify.com/install-cli.sh | bash
 irm https://apify.com/install-cli.ps1 | iex
 ```
 
-## Other methods
+:::tip No need for Node.js
+
+If you install Apify CLI using our installation scripts, you don't need to install Node.js to use the CLI. This is because we use [Bun](https://bun.sh/) as a bundler, which creates a standalone executable file. This is especially useful for Python users or anyone who prefers not to manage Node.js dependencies.
+
+:::
 
-### Homebrew
+## Homebrew
 
 ```bash
 brew install apify-cli
 ```
 
-### NPM
+:::tip Node.js required with Homebrew
+
+If you install Apify CLI using Homebrew, be aware that internally it still installs Node.js, as Homebrew does not support having everything in one executable file.
```

Comment:
```suggestion
If you install Apify CLI using Homebrew, be aware that internally it will also install Node.js. This should not cause issues, but if it does, you might need to adjust your `PATH` environment variable to deprioritize Homebrew's install of Node.js
```